ledger/ledgercore/accountresource.go

```diff
@@ -28,6 +28,34 @@ type AccountResource struct {
 	AppParams     *basics.AppParams
 }
 
+// AccountResourceModel used to encode AccountResource
+type AccountResourceModel struct {
```

Comment:
this need to be "transplanted" into `/go-algorand/daemon/algod/api/spec/v2/model.go`, since it's a UI model only.